src/type/definition.ts

```diff
@@ -1724,14 +1701,46 @@ export interface GraphQLInputFieldConfig {
 
 export type GraphQLInputFieldConfigMap = ObjMap<GraphQLInputFieldConfig>;
 
-export interface GraphQLInputField {
+export class GraphQLInputField extends GraphQLSchemaElement {
```

Comment:
Does this become a breaking change? If folks were importing `GraphQLInputField` the interface - which may be different to `GraphQLInputField` the class?

Reply:
Actually I guess not, if even the function below still works:

```
export function isRequiredInputField(field: GraphQLInputField): boolean
```

I guess TS treats classes and interfaces the same here